### PR TITLE
Safe Area and FloatingNavigationButton improvements.

### PR DESCRIPTION
**Pull request by haavamoa**
### Description of Change

- [Layout] Made sure all layouts ignore safe area when its not set by the consumer. Grid, StackLayout etc.
- [FloatingNavigationButton] Made sure people can tap the background of the FAB to close it.
- [FloatingNavigationButton] Made sure sizes of the icons are smaller for better UX.

### Todos
- [X] I have tested on an Android device.
- [X] I have tested on an iOS device.
- [ ] I have supported accessibility

### CODE REVIEW

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs [hunk 1]
```diff
@@ -25,23 +25,34 @@ public FloatingNavigationButton(FloatingNavigationButtonConfigurator floatingNav
...
-        m_contentGrid.CascadeInputTransparent = false;
-        m_contentGrid.InputTransparent = true;
+        // m_contentGrid.CascadeInputTransparent = false;
 
         AddMainButton();
         CreateAnimations();
+        Loaded += OnLoaded;
     }
+
+    private void OnLoaded(object? sender, EventArgs e)
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:40]: Bruk OnHandlerChanging istedet ? 😇

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs [hunk 2]
```diff
@@ -25,23 +25,34 @@ public FloatingNavigationButton(FloatingNavigationButtonConfigurator floatingNav
         Add(m_contentGrid);
         
         Padding = new Thickness(0, 0, Sizes.GetSize(SizeName.size_3), Sizes.GetSize(SizeName.size_13));
-        CascadeInputTransparent = false;
+        // CascadeInputTransparent = false;
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:28]: Fjern

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs [hunk 3]
```diff
@@ -25,23 +25,34 @@ public FloatingNavigationButton(FloatingNavigationButtonConfigurator floatingNav
...
         
         m_contentGrid.RowDefinitions = new RowDefinitionCollection { new() { Height = GridLength.Star } };
         m_contentGrid.ColumnDefinitions = new ColumnDefinitionCollection { new() { Width = GridLength.Auto } };
         m_contentGrid.HorizontalOptions = LayoutOptions.End;
-        m_contentGrid.CascadeInputTransparent = false;
-        m_contentGrid.InputTransparent = true;
+        // m_contentGrid.CascadeInputTransparent = false;
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:33]: Fjern

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs [hunk 4]
```diff
@@ -25,23 +25,34 @@ public FloatingNavigationButton(FloatingNavigationButtonConfigurator floatingNav
...
+    }
+    
     
-    protected override void OnHandlerChanged()
+    private void MakeBackgroundClickable()
     {
-        base.OnHandlerChanged();
-        
-        InputTransparent = true;
+        // InputTransparent = false;
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:54]: Fjern

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs [hunk 5]
```diff
@@ -148,16 +149,18 @@ private void Expand()
...
     }
 
+
     public async Task Close()
     {
         if(!m_isExpanded)
             return;
         
-        InputTransparent = true;
+        // InputTransparent = true;
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:158]: Fjern

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/NavigationMenuButton/NavigationMenuButton.cs [hunk 6]
```diff
@@ -12,35 +13,29 @@ public NavigationMenuButton()
         RowDefinitions = new RowDefinitionCollection { new() { Height = GridLength.Auto } };
         ColumnDefinitions = new ColumnDefinitionCollection { new() { Width = GridLength.Auto } };
         
-        ImageButton = new Images.ImageButton.ImageButton()
+        Button = new Button()
         {
             BorderColor = Colors.GetColor(ColorName.color_system_white),
             BorderWidth = 3,
-            TintColor = Colors.GetColor(ColorName.color_system_white)
+            // TintColor = Colors.GetColor(ColorName.color_system_white) TODO: LEGG TIL ImageColor på Button
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/NavigationMenuButton/NavigationMenuButton.cs:20]: Gjør dæ ?

**Review by Vetle444 — CHANGES_REQUESTED** [review]

File: src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs (hunk 1 above)
Comment (haavamoa) [reply src/library/DIPS.Mobile.UI/Components/Navigation/FloatingNavigationButton/FloatingNavigationButton.cs:40]: Er ikke sikkert jeg trenger å gjøre dette i det hele tatt, skal sjekke.

**Review by haavamoa — COMMENTED** [review]
Må fikse at man må trykke to ganger for å lukke den på Android, merket under testing.

**Review by Vetle444 — APPROVED** [review]

**Pull request merged**